groups/reconcile.go

```diff
@@ -221,53 +222,66 @@ func NewReconciler(ctx context.Context, clientOption option.ClientOption) (*Reco
 }
 
 func (r *Reconciler) ReconcileGroups(groups []GoogleGroup) error {
+	// aggregate the errors that occured and return them together in the end.
+	var errs []error
 	for _, g := range groups {
 		if g.EmailId == "" {
-			return fmt.Errorf("group has no email-id: %#v", g)
+			errs = append(errs, fmt.Errorf("group has no email-id: %#v", g))
+			continue
 		}
 
 		err := r.AdminSvc.CreateOrUpdateGroupIfNescessary(g)
 		if err != nil {
-			return err
+			errs = append(errs, err)
+			continue
```

Comment:
I would prefer to drop the continues from here on. One bad member should not stop attempts to add the rest, etc

Reply:
ahh yes, thanks for catching that